### ISSUE TITLE
Printing training accuracy

### ISSUE THREAD

**Issue by tiny-dentist** [alanpurple/hgru4rec_tf#1]
Hi, is it possible to print the training accuracy ?

**Comment by alanpurple** [comment]
@rahulsingh701 

of course, but this repo is bit of an old one.....so it takes time to remind

**Comment by tiny-dentist** [comment]
@alanpurple can you please tell me how will I be able to achieve this?
